src/client/components/select/stories/select.stories.tsx

```diff
@@ -12,21 +13,16 @@ const actions = {
   onChange: action('onChange'),
 }
 
-const Container = ({ children }: { children: any }) => {
-  return <div style={{ maxWidth: '320px', fontFamily: 'Arial, sans-serif' }}>
-    {children}
-  </div>
-}
+const container = createContainerDecorator({ maxWidth: '350px' })
 
 storiesOf('components.select', module)
+  .addDecorator(container)
```

Comment:
I think this one is too specific to be generic, can we just inline the decorator here? e.g.

```ts
.addDecorator((story: RenderFunction) => {
  return <div style={{ maxWidth: 350 }}>{story()}</div>
})
```

Reply:
Yeah, makes sense.

Reply:
Done!